languages/nim/maintainers.md

```diff
@@ -8,4 +8,4 @@ TODO: add senior maintainers
 
 ## Contributing Maintainers
 
-TODO: add contributing maintainers
+Yoni (Slack: `@ynfle`, GitHub: [ynfle](https://github.com/ynfle))
```

Comment:
```suggestion
- Yoni (Slack: `@ynfle`, GitHub: [ynfle](https://github.com/ynfle))
```

The convention seems to be "use `-`, even for single-item maintainer lists". See:
- [bash/maintainers.md](https://github.com/exercism/v3/blob/4c637f68be11/languages/bash/maintainers.md)
- [dart/maintainers.md](https://github.com/exercism/v3/blob/4c637f68be11/languages/dart/maintainers.md)
- [delphi/maintainers.md](https://github.com/exercism/v3/blob/4c637f68be11/languages/delphi/maintainers.md)
- [elm/maintainers.md](https://github.com/exercism/v3/blob/4c637f68be11/languages/elm/maintainers.md)
- [erlang/maintainers.md](https://github.com/exercism/v3/blob/4c637f68be11/languages/erlang/maintainers.md)
- [j/maintainers.md](https://github.com/exercism/v3/blob/4c637f68be11/languages/j/maintainers.md)
- [perl5/maintainers.md](https://github.com/exercism/v3/blob/4c637f68be11/languages/perl5/maintainers.md)
- [purescript/maintainers.md](https://github.com/exercism/v3/blob/4c637f68be11/languages/purescript/maintainers.md)
- [typescript/maintainers.md](https://github.com/exercism/v3/blob/4c637f68be11/languages/typescript/maintainers.md)
- [x86-64-assembly/maintainers.md](https://github.com/exercism/v3/blob/4c637f68be11/languages/x86-64-assembly/maintainers.md)